crates/matrix-sdk-sqlite/src/lib.rs

```diff
@@ -43,13 +44,21 @@ pub use self::state_store::{SqliteStateStore, DATABASE_NAME as STATE_STORE_DATAB
 #[cfg(test)]
 matrix_sdk_test_utils::init_tracing_for_tests!();
 
+#[derive(Clone, Debug, Eq, PartialEq, Zeroize)]
+pub enum Secret {
+    #[zeroize]
+    Key([u8; 32]),
+    #[zeroize]
+    PassPhrase(String),
```

Comment:
Same here, this will produce unintended copies, `Zeroizing<String>` would make sense.